bridge/irc_manager.go

```diff
@@ -333,14 +344,16 @@ func (m *IRCManager) SendMessage(channel string, msg *DiscordMessage) {
 
 	// Person is appearing offline (or the bridge is running in Simple Mode)
 	if !ok {
-		length := len(msg.Author.Username)
+		// length := len(msg.Author.Username)
 		for _, line := range strings.Split(content, "\n") {
-			m.bridge.ircListener.Privmsg(channel, fmt.Sprintf(
-				"<%s#%s> %s",
-				msg.Author.Username[:1]+"\u200B"+msg.Author.Username[1:length],
-				msg.Author.Discriminator,
-				line,
-			))
+			// m.bridge.ircListener.Privmsg(channel, fmt.Sprintf(
+			// 	"<%s#%s> %s",
+			// 	msg.Author.Username[:1]+"\u200B"+msg.Author.Username[1:length],
+			// 	msg.Author.Discriminator,
+			// 	line,
+			// ))
```

Comment:
You can just delete this code instead of commenting out

Reply:
Oops, I accidentally left it there, was what I refered to and forgot to remove, will pull those comments out.